### DISCUSSION TITLE
Is there any risk with setting the home directory to writable?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [lima-vm/lima#550] [Q&A] [answered]
Would there be any issues with running linux on your Mac home directory? Like could linux apps you're using write to the home directory and cause issues by overwriting important Mac files?

**Comment by jandubois** [comment]
> Like could linux apps you're using write to the home directory and cause issues by overwriting important Mac files?

I'm not sure I understand this question: why would you want to make the directory writable unless you want your Linux apps to write to it?

**Reply by YousufSSyed** [reply]
I mean, what are the odds of a Linux app writing data to the home directory without you writing a command or somehow explicitly telling it to write to the home directory (like if the app automatically writes app data to the directory), that overwrites or messes with macOS or Mac app data?

**Comment by AkihiroSuda** [comment, marked as answer]
Writing home is probably safe in most cases, but if you attempt to modify a same file from macOS and Linux simultaneously you may get corrupted files, so it is read-only by default
